changelog/277.feature.rst

```diff
@@ -0,0 +1,7 @@
+|commands| Add a way to get the parent or root commands of slash commands.
+- Add :attr:`SubCommand.parent <ext.commands.SubCommand.parent>`
+- Add :attr:`SubCommandGroup.parent <ext.commands.SubCommandGroup.parent>`
+- Add :attr:`SubCommand.root_command <ext.commands.SubCommand.root_command>` property
+- Add :attr:`SubCommandGroup.root_command <ext.commands.SubCommandGroup.root_command>` property
+- Add a new required argument (``parent``) to :attr:`SubCommand <ext.commands.SubCommand>`
+- Add a new required argument (``parent``) to :attr:`SubCommandGroup <ext.commands.SubCommandGroup>`
```

Comment:
these two should be part of a different file, `759.breaking.rst`

Reply:
Done